### DISCUSSION TITLE
how Trigger.dev work?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [triggerdotdev/trigger.dev#1082] [Q&A] [answered]
hi, i just found Trigger and very like the idea "no timeout" but how exactly it work? 
lets me give you an example. 
I need to download a large file on server from an Api it take up to 5 minutes to complete. How can Trigger by pass Vercel 30s for this task? 
thank you for any answers.

**Comment by matt-aitken** [comment, marked as answer]
If you use v3 then the work is offloaded to separate long-running servers with no timeouts. For v2, which is what you're talking about here, the work is split into smaller chunks using `io.runTask()` inside your Job. This won't for your use case though because you can't divide a fetch up.

This would work with v3, drop by [our Discord](https://trigger.dev/discord) and message us for access.
